GUI/QtQuick/qml/Settings/ModifyArmoryServer.qml

```diff
@@ -0,0 +1,202 @@
+            Layout.alignment: Qt.AlignVCenter
+
+            width: 126
+            height: 19
+
+            color: "#E2E7FF"
+            font.pixelSize: 16
+            font.family: "Roboto"
+            font.weight: Font.Normal
+        }
+
+        CustomRadioButton {
+            id: radbut_main
+
+            Layout.alignment: Qt.AlignVCenter
+
+            text: "MainNet"
+
+            spacing: 6
+            font.pixelSize: 13
+            font.family: "Roboto"
+            font.weight: Font.Normal
+
+            checked: true
+
+            // netType==0 => MainNet, netType==1 => TestNet
+            onClicked : {
+                armoryServersModel.setData(armoryServersModel.index(server_index, 0)
+                                           , 0, ArmoryServersModel.NetTypeRole)
+            }
+        }
+
+        CustomRadioButton {
+            id: radbut_test
+
+            Layout.alignment: Qt.AlignVCenter
+
+            text: "TestNet"
+
+            spacing: 6
+            font.pixelSize: 13
+            font.family: "Roboto"
+            font.weight: Font.Normal
+
+            checked: false
+
+            // netType==0 => MainNet, netType==1 => TestNet
+            onClicked : {
+                armoryServersModel.setData(armoryServersModel.index(server_index, 0)
+                                           , 1, ArmoryServersModel.NetTypeRole)
+            }
+        }
+
+        Label {
+            Layout.fillWidth: true
+        }
+    }
+
+    CustomTextInput {
+        id: name_text_input
+
+        Layout.alignment: Qt.AlignTop | Qt.AlignHCenter
+        Layout.preferredHeight : 70
+        Layout.preferredWidth: 532
+        Layout.topMargin: 10
+
+        title_text: qsTr("Name")
+
+        onTextEdited: {
+            armoryServersModel.setData(armoryServersModel.index(server_index, 0)
+                                       , name_text_input.input_text, ArmoryServersModel.NameRole)
+        }
+    }
+
+    CustomTextInput {
+        id: ip_dns_text_input
+
+        Layout.alignment: Qt.AlignTop | Qt.AlignHCenter
+        Layout.preferredHeight : 70
+        Layout.preferredWidth: 532
+        Layout.topMargin: 10
+
+        title_text: qsTr("IP/DNS")
+
+        input_validator: RegExpValidator { regExp: /(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)(\.(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)){3}(,(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)(\.(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)){3})*/ }
+
+        onTextEdited: {
+            armoryServersModel.setData(armoryServersModel.index(server_index, 0)
+                                       , ip_dns_text_input.input_text, ArmoryServersModel.AddressRole)
+        }
+    }
+
+    CustomTextInput {
+        id: port_text_input
+
+        Layout.alignment: Qt.AlignTop | Qt.AlignHCenter
+        Layout.preferredHeight : 70
+        Layout.preferredWidth: 532
+        Layout.topMargin: 10
+
+        title_text: qsTr("Port")
+
+        input_validator: IntValidator {bottom: 0; top: 65536;}
```

Comment:
Port number also can't be 0 and 65536, the range should be 1-65535, or better 80-65535